src/ontology/cl-edit.owl

```diff
@@ -7755,6 +7756,7 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000508 "FMA:67609")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000508 "G cell")
 AnnotationAssertion(rdfs:label obo:CL_0000508 "type G enteroendocrine cell")
```

Comment:
Type G enteroendocrine cell is not exclusively found in the stomach, it is also found in duodenum and pancreas (https://en.wikipedia.org/wiki/G_cell). It has been inferred that `Type G enteroendocrine cell ` is a subclass of` stomach neuroendocrine cell`.  

To remain consistent with the other patterns, I would suggest keeping `Type G enteroendocrine cell ` as a broad term (e.g. `enteroendocrine cell `and `neuroendocrine cell `and `gastrin secreting cell ` and `enkephalin secreting cell`.

Then create a specific term `type G cell of stomach` that is ` type G enteroendocrine cell`
 and `('part of' some 'epithelium of stomach')` or `'part of' some 'mucosa of pyloric antrum'`

Reply:
Updated definition to reflect location and created new term for type g cell of stomach